eng/pipeline-pr.yml

```diff
@@ -191,19 +193,24 @@ jobs:
           $(_PlatformArgs)
           $(_InternalRuntimeDownloadArgs)
           /bl:$(Build.SourcesDirectory)\artifacts\log\$(_BuildConfig)\Test.binlog
+          /p:Coverage=$(_Coverage)
         displayName: Run xUnit Tests
-        condition: and(or(ne(variables['_HelixPipeline'], 'true'), and(eq(variables['_HelixPipeline'], 'true') ,eq(variables['_BuildConfig'], 'Release'), eq(variables['_PublicBuildPipeline'], 'true'), eq(variables['_ContinuousIntegrationTestsEnabled'], 'true'))), ne(variables['_Platform'], 'arm64'))
+        continueOnError: true
```

Comment:
Does this mean that even if the test fails the pipeline won't show any failures ?

Reply:
The next step would execute even if this step failed. In case of failures, the pipeline would show Orange color signifying some steps failed. We need to continue because we need to publish the test results even on failures, for feedback to devs.